lib/fs.js

```diff
@@ -376,6 +376,16 @@ fs.exists = function(path, callback) {
   }
 };
 
+Object.defineProperty(fs.exists, internalUtil.promisify.custom, {
+  enumerable: false,
```

Comment:
Indeed, I was copying and pasting from line 780, because I incorrectly assumed that `enumerable: false` was required to conceal it from userland.

MDN says the defaults are:

```{enumerable: false, configurable: false, writeable: false}```

Those look like good defaults to me. I'll remove `enumerable: false` here and use the defaults. (I guess we should remove it at line 780, but I'll let somebody else do that.)